enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/jwt/UnsecuredAPIAuthenticator.java

```diff
@@ -79,7 +86,29 @@ public AuthenticationContext authenticate(RequestContext requestContext) throws
                 throw new APISecurityException(APIConstants.StatusCodes.SERVICE_UNAVAILABLE.getCode(),
                         GeneralErrorCodeConstants.API_BLOCKED_CODE, GeneralErrorCodeConstants.API_BLOCKED_MESSAGE);
             }
-            return FilterUtils.generateAuthenticationContextForUnsecured(requestContext);
+
+            // Generate or get backend JWT
+            JWTConfigurationDto backendJwtConfig = ConfigHolder.getInstance().getConfig().
+                    getJwtConfigurationDto();
+            String endUserToken = null;
+
+            if (backendJwtConfig.isEnabled()) {
+                JWTValidationInfo validationInfo = new JWTValidationInfo();
```

Comment:
we can just pass a `null` or simply pass `new  JWTValidationInfo()` when we are calling `generateJWTInfoDto` since we don't use the reference to this object anywhere else.